client/src/App.js

```diff
@@ -1,14 +1,19 @@
 import React, { Component } from 'react';
 import Nav from './components/nav/nav';
+import SearchBar from './components/searchbar/searchbar.js';
 import Footer from './components/footer/footer';
 
 class App extends Component {
 	render() {
 		return (
-			<div className='App'>
+			<div className="App">
 				<Nav />
```

Comment:
Indenting seems a little bit off here, but it may just be because of the extra components. Otherwise, looks good =)